guides/source/form_helpers.md

```diff
@@ -233,10 +231,10 @@ For these helpers the first argument is the name of an instance variable and the
 will produce output similar to
 
 ```erb
-<input id="person_name" name="person[name]" type="text" value="Henry"/>
+<input id="person_name" name="person[name]" type="text" value="Henry" />
 ```
 
-Upon form submission the value entered by the user will be stored in `params[:person][:name]`. The `params[:person]` hash is suitable for passing to `Person.new` or, if `@person` is an instance of Person, `@person.update`. While the name of an attribute is the most common second parameter to these helpers this is not compulsory. In the example above, as long as person objects have a `name` and a `name=` method Rails will be happy.
```

Comment:
Why is this useless?

Reply:
We have another place where we completly explain how to use `params`, see https://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters